src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/ValidateWindowsAccount.cs

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Migration.Contracts
+{
+    public class ValidateWindowsAccountRequestParams
+    {
+        public string Username { get; set; }
+        public string Password { get; set; }
+    }
+
+    public class ValidateWindowsAccountResult
+    {
+        public bool Success { get; set; }
```

Comment:
As mentioned below - we have SendError for if the request wasn't able to be completed. So generally I would say that these aren't necessary - if the request completes without throwing then it means it passed.